projects/aca-playwright-shared/src/page-objects/pages/favorites.page.ts

```diff
@@ -44,8 +44,15 @@ export class FavoritesPage extends BasePage {
   public sidenav = new SidenavComponent(this.page);
   public pagination = new PaginationComponent(this.page);
   public breadcrumb = new Breadcrumb(this.page);
+  public infoDrawer = new AdfInfoDrawerComponent(this.page);
 
   async waitForPageLoad() {
     await this.page.waitForURL(`**/${FavoritesPage.pageUrl}`);
   }
+
+  async addCommentToNode(commentText: any) {
```

Comment:
Please don't use `any` we should always type variables when possible

Reply:
WIll be fixed in the next push